exporter/datadogexporter/factory.go

```diff
@@ -53,6 +53,12 @@ var metricExportNativeClientFeatureGate = featuregate.GlobalRegistry().MustRegis
 	featuregate.WithRegisterDescription("When enabled, metric export in datadogexporter uses native Datadog client APIs instead of Zorkian APIs."),
 )
 
+var metricRemappingEnabledFeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"exporter.datadogexporter.metricremappingenabled",
+	featuregate.StageBeta,
```

Comment:
This is confusing to me, if we are looking to eventually disable this feature gate, then we can't start at beta, that's not the lifecycle of a feature gate according to https://github.com/open-telemetry/opentelemetry-collector/tree/main/featuregate#feature-lifecycle

Reply:
Maybe rename this feature gate to `disablemetricremapping` and mark it alpha?